modin/core/dataframe/pandas/partitioning/partition_manager.py

```diff
@@ -1608,5 +1610,5 @@ def shuffle_partitions(
         else:
             # If there are not pivots we can simply apply the function row-wise
             return np.array(
-                [[row_part.apply(final_shuffle_func)] for row_part in row_partitions]
+                [row_part.apply(final_shuffle_func) for row_part in row_partitions]
```

Comment:
`row_part` is now actually a row partition returning a list, meaning there's no need to wrap this into a list no more

Reply:
Makes sense!